lib/middleware.ts

```diff
@@ -68,6 +69,22 @@ function getIp(req) {
   return req.ip || remoteAddress;
 }
 
+/**
+ * Log level
+ */
+function getLogLevel(req, res) {
+  const statusCode = res.statusCode;
+  let result;
+  if (statusCode >= 500) {
+    result = kayveeLogger.Error;
+  } else if (statusCode >= 400) {
```

Comment:
This was just the strawman from the spec:

`Strawman: 2xx, 3xx is info, 4xx is notice, 499, 5xx is error`